### PR TITLE
zh-rCN i18n initialization

### PR DESCRIPTION
**Pull request by HankAviator**
Simplified Chinese localisation

### CODE REVIEW

**Comment by peterjosling** [comment]
Can you remove all the new files except strings.xml?

Thanks for this! 😃

**Comment by HankAviator** [comment]
Done 😄

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<resources>
+    <string name="app_name">Scroball</string>
+
+    <!-- Strings related to Settings -->
+    <string name="saved_session_key">会话密钥</string>
```
Comment (peterjosling) [inline app/src/main/res/values-zh-rCN/strings.xml]: This needs to be left as sessionKey as well (it's for storage, not localisation).

**Comment by peterjosling** [comment]
Thanks! 😄

**Pull request merged**